src/modules/contact/components/form/style.module.css

```diff
@@ -0,0 +1,24 @@
+/* Remove number input default arrow style */
+.phone::-webkit-inner-spin-button {
+    appearance: none;
+}
+
+.sendBtn{
+    background-color: var(--layout-secondary);
+    border: none;
+    color: var(--layout-primary);
+    padding: 15px 32px;
+    text-align: center;
+    outline: none;
+    cursor: pointer;
+    transition-duration: 0.2s;
+}
+
+.sendBtn:hover {
```

Comment:
Support the `focus` also to improve accessibility.

Reply:
> Support the `focus` also to improve accessibility.

`active` already exist, `focus` wont mean much.

Reply:
The support for `active` does not improve the accessibility, try to navigate with the `tab` key and you'll notice that.